warrior/Actions/CiRegressionActions/ci_regression_actions.py

```diff
@@ -329,3 +329,99 @@ def delete_tmp_dir(self):
         """
         path = data_Utils.get_object_from_datarepository("parallel_exec_tmp_dir")
         return file_Utils.delFolder(path)
+
+    def check_kw_arg_type(self, str_value, int_value, float_value, bool_value,
```

Comment:
If check_type in check_values_from_datafile can achieve the same test coverage, then we don't need 2 functions

Reply:
check_type is for arguments starting with type_ (like int_, file_ etc.,) whereas the other is wtag units. I have moved the changes from war-1172 to here, so that there is less confusion.

Reply:
I see, please update the documentation for this keyword to clarify:
This keyword is intended to test the type prefix for keyword arguments
when an argument name has a type_ prefix, the variable type will become the type specified in the type_ prefix

Reply:
Also the keyword name is confusing to me
it'd be clearer if it says check_kw_arg_type_prefix

Reply:
done.